api/src/main/java/org/openmrs/TagConstants.java

```diff
@@ -19,4 +19,8 @@
 	public final static String GET_TAGS = "Get Tags";
 	
 	public final static String VIEW_OBJECTS = "View Objects";
+	
+	public final static String TAG_UUID = "7379a456-296a-2331-9c83-a3714b82d7ac";
```

Comment:
classifier is just a logic text you add to the artifact name, you should be adding the test jar instead with no classifier